portal-web/test/functional/com/liferay/portalweb/macros/headless/blogs/HeadlessBlog.macro

```diff
@@ -30,4 +31,64 @@ definition {
 		return "${entryId}";
 	}
 
+	macro assertCommentERCWithCorrectValue {
+		var actualCommentERCValue = JSONUtil.getWithJSONPath("${response}", "$..['items'][?(@.creator)].externalReferenceCode");
+
+		TestUtils.assertEquals(
+			actual = "${actualCommentERCValue}",
+			expected = "${expectedCommentERCValue}");
+	}
+
+	macro assertCommentTextWithCorrectValue {
```

Comment:
same here, try to create a generic method for all the values of the body

Reply:
Done!